crates/goose-cli/src/commands/configure.rs

```diff
@@ -166,9 +213,251 @@ pub fn get_required_keys(provider_name: &str) -> Vec<&'static str> {
         "openai" => vec!["OPENAI_API_KEY"],
         "databricks" => vec!["DATABRICKS_HOST"],
         "ollama" => vec!["OLLAMA_HOST"],
-        "anthropic" => vec!["ANTHROPIC_API_KEY"], // Removed ANTHROPIC_HOST since we use a fixed endpoint
+        "anthropic" => vec!["ANTHROPIC_API_KEY"],
         "google" => vec!["GOOGLE_API_KEY"],
         "groq" => vec!["GROQ_API_KEY"],
         _ => panic!("Invalid provider name"),
     }
 }
+
+/// Configure systems that can be used with goose
+/// Dialog for toggling which systems are enabled/disabled
+pub fn toggle_systems_dialog() -> Result<(), Box<dyn Error>> {
+    // Load existing config
+    let mut config = Config::load().unwrap_or_default();
+
+    if config.systems.is_empty() {
+        let _ = cliclack::outro(
+            "No systems configured yet. Run configure and add some systems first.",
+        )?;
+        return Ok(());
+    }
+
+    // Create a list of system names and their enabled status
+    let mut system_status: Vec<(String, bool)> = Vec::new();
+    for (name, entry) in config.systems.iter() {
+        system_status.push((name.clone(), entry.enabled));
+    }
+
+    // Get currently enabled systems for the selection
+    let enabled_systems: Vec<&String> = system_status
+        .iter()
+        .filter(|(_, enabled)| *enabled)
+        .map(|(name, _)| name)
+        .collect();
+
+    // Let user toggle systems
+    let selected =
+        cliclack::multiselect("enable systems: (use \"space\" to toggle and \"enter\" to submit)")
+            .items(
+                &system_status
+                    .iter()
+                    .map(|(name, _)| (name, name.as_str(), ""))
+                    .collect::<Vec<_>>(),
+            )
+            .initial_values(enabled_systems)
+            .interact()?;
```

Comment:
I can't disable all my systems, if thats intentional we should add `at least one must be enabled`. "Input required" wasn't clear to me.

![image](https://github.com/user-attachments/assets/9c621320-504f-4d49-8e59-03cd36b4a810)

Reply:
fixing! just missed an option on the multiselect